test/infrastructure/docker/docker/loadbalancer.go

```diff
@@ -59,6 +61,7 @@ func NewLoadBalancer(name string) (*LoadBalancer, error) {
 	return &LoadBalancer{
 		name:      name,
 		container: container,
+		ipv6:      ipv6,
```

Comment:
Might be work to keep IpFamily here (it depends by what is the expected behaviour for dual mode clusters)